cfme/services/catalogs/catalog_item.py

```diff
@@ -1,107 +1,192 @@
+        )
+
+
+class EditCatalogItemView(BasicInfoForm):
+
+    save_button = Button('Save')
+    reset_button = Button('Reset')
+
+    @property
+    def is_displayed(self):
+        return (
+            self.in_explorer and self.catalog_items.is_opened and
+            self.title.text == "Editing Service Catalog Item {}".format(self.name)
+        )
+
+
+class AddButtongroupView(ButtonGroupForm):
+
+    add_button = Button("Add")
+
+    @property
+    def is_displayed(self):
+        return (
+            self.in_explorer and self.catalog_items.is_opened and
+            self.title.text == "Adding a new Button Group"
+        )
+
+
+class AddButtonView(ButtonForm):
+
+    add_button = Button("Add")
+
+    @property
+    def is_displayed(self):
+        return (
+            self.in_explorer and self.catalog_items.is_opened and
+            self.title.text == "Adding a new Button"
+        )
+
+
+class EditTagsView(EditTagsForm):
+
+    save_button = Button('Save')
+    add_button = Button("Add")
+
+    @property
+    def is_displayed(self):
+        return (
+            self.in_explorer and self.catalog_items.is_opened and
+            self.title.text == 'Editing My Company Tags for "Service Catalog Items"'
+        )
+
+
+class AddCatalogBundleView(BasicInfoForm):
+    apply_button = Button('Apply')
+    add_button = Button("Add")
+
+    @property
+    def is_displayed(self):
+        return (
+            self.in_explorer and self.catalog_items.is_opened and
+            self.title.text == "Adding a new Catalog Bundle"
+        )
+
+
+class EditCatalogBundleView(BasicInfoForm):
+
+    save_button = Button('Save')
+    reset_button = Button('Reset')
+
+    @property
+    def is_displayed(self):
+        return (
+            self.in_explorer and self.catalog_items.is_opened and
+            self.title.text == 'Editing Catalog Bundle "{}"'.format(self.obj.name)
+        )
 
 
 class CatalogItem(Updateable, Pretty, Navigatable):
-    pretty_attrs = ['name', 'item_type', 'catalog', 'catalog_name', 'provider', 'domain']
 
-    def __init__(self, item_type=None, vm_name=None, name=None, description=None,
-                 display_in=False, catalog=None, dialog=None,
+    def __init__(self, name=None, description=None, items=None, item_type=None,
+                 vm_name=None, display_in=False, catalog=None, dialog=None,
                  catalog_name=None, orch_template=None, provider_type=None,
-                 provider=None, config_template=None, prov_data=None, domain="ManageIQ (Locked)",
-                 appliance=None):
-        self.item_type = item_type
-        self.vm_name = vm_name
+                 provider=None, config_template=None, prov_data=None,
+                 domain="ManageIQ (Locked)", appliance=None):
+        Navigatable.__init__(self, appliance=appliance)
         self.name = name
         self.description = description
+        self.item_type = item_type
```

Comment:
**Required**
What is items param here for? I don't see it used anywhere else and its not set as an object attribute